ev_charge/lib/database/database.dart

```diff
@@ -1,13 +1,14 @@
 import 'package:drift/drift.dart';
 import 'package:drift_flutter/drift_flutter.dart';
+import 'package:flutter/foundation.dart';
 import 'package:path_provider/path_provider.dart';
 
 part 'database.g.dart';
 
 class EVCarModels extends Table {
   late final id = integer().autoIncrement()();
   late final modelName = text().withLength(min: 3, max: 64)();
-  late final modelYear = integer()();
+  late final modelYear = integer().nullable()();
```

Comment:
```suggestion
  late final modelYear = integer()();
```
I følge backenden skal det her ikke være nullable og jeg kan ikke se grunden til at det pludseligt skuller være det

Reply:
Var lavet inden databasen på backenden, men har ændret til at de følger hinanden